scripts/opentitan/make_release.sh

```diff
@@ -18,4 +18,4 @@ QEMU_BUILD="$3"
 tar --create --auto-compress --verbose --file="$OUT_TARBALL" \
     --directory="$QEMU_DIR" \
     "$QEMU_BUILD"/qemu-{system-riscv32,img} \
-    scripts/opentitan/{otpconv,flashgen}.py
+    scripts/opentitan/{otptool,flashgen}.py
```

Comment:
`otptool.py` depends on many other Python modules which are located in `python/qemu/ot...`.
I'm not the author of this script, I'm not sure about its purposes but is it ok to put the script into the tarball but ignore its dependencies?

Reply:
Good catch, I've included that directory